src/pages/ProductPage/index.scss

```diff
@@ -19,47 +19,47 @@
       }
   }
 
-  .gallery{
+  .gallery {
 
-    button{
+    .button {
       position: absolute;
       border-radius: 25px;
     }
 
-    .button__icon--previous{
+    .button__icon--previous {
       z-index: 99;
       left: 10px;
       top: 175px;
     }
 
-    .button__icon--next{
+    .button__icon--next {
       z-index: 99;
       top: 175px;
       left: 345px;
     }
 
-    img{
+    & .__image {

```

Comment:
In BEM you never start with `__` only use it so separate one module to a block `galley__item` (module__block).

Take a look at this [doc](http://getbem.com/naming/) about how to naming.